### ISSUE TITLE
make a cleaner division between lesson design & Workbench use

### ISSUE THREAD

**Issue by tobyhodges** [carpentries/lesson-development-training#97]
Based on feedback, trainer experience, and discussions during the first pilot, we might try more clearly separating the lesson design elements of the curriculum from the Workbench/building a lesson site parts. These two types of task are intentionally mixed together in the current version of the curriculum, but we felt there was enough evidence of split attention effect/cognitive load from context switching that it might be worth trying out a more clean split, e.g. lesson design on the first full day, Workbench, how we operate, and preparing to teach on the second full day.

This approach could be supported by getting participants to put their notes, objectives, exercises, etc into a single CodiMD (created from the Lesson Design Notes template, see #72). Once they are oriented with the Workbench, we could then make exercises and follow-along examples to copy/paste most of this out in chunks, into the relevant parts of the lesson repository.

**Comment by tobyhodges** [comment]
assigning this one to me and @anenadic as she was interested in tackling it too.

**Comment by sstevens2** [comment]
I agree with Erin about the Objectives going together instead of having the workbench between them. I'm otherwise hesitant to wait to introduce the workbench any longer as it gives them a place to put their work and gets them starting to make progress on the lesson webpage early on.

**Comment by sstevens2** [comment]
We've also added #371 related to this issue.

The maintainers have discussed and we prefer to close this issue and keep the workbench use integrated into the lesson.  We've decided that at the moment we would prefer to have learners using the workbench right away so they have a realistic experience of getting started putting their work directly into their lesson webpage and their history of work on the lesson is starting to be recorded.
